coderd/prometheusmetrics/aggregator_test.go

```diff
@@ -119,6 +124,10 @@ func verifyCollectedMetrics(t *testing.T, expected []agentsdk.AgentMetric, actua
 		}
 
 		dtoLabels := asMetricAgentLabels(d.GetLabel())
+		// dto labels are sorted in alphabetical order.
```

Comment:
The expected labels that we pass in have to be sorted. The DTO ones are already sorted.
I figured it was fine in the `verify` function to sort them, otherwise maybe we would have to move it closer to where we define `expected`?

https://github.com/coder/coder/blob/8bfa9ad8a5df033cca717e2024962f3ffd9045dd/coderd/prometheusmetrics/aggregator_test.go#L66-L79

But this only needs to be sorted when comparing to dto labels.

Reply:
I see now what you mean. I guess we can leave it as is, there won't be a huge gain if you refactor it 👍